test/e2e/helpers.go

```
@@ -0,0 +1,811 @@
+		actualProtocol := corev1.ProtocolTCP
+		if p.Protocol != nil {
+			actualProtocol = *p.Protocol
+		}
+		if actualProtocol == protocol {
+			return true
+		}
+	}
+	return false
+}
+
+func hasIngressFromNamespace(rules []networkingv1.NetworkPolicyIngressRule, port int32, namespace string) bool {
```

golangci-lint (2.11.4)

[error] func hasIngressFromNamespace is unused

(unused)